Makefile

```diff
@@ -207,6 +207,7 @@ qgc_firmware: \
 	check_mindpx-v2_default \
 	check_px4fmu-v1_default \
 	check_px4fmu-v2_default \
+	check_px4fmu-v2_lpe \
```

Comment:
Everything under `qgc_firmware` is built on travis-ci and pushed to s3 for master/beta/stable.